### PR TITLE
Update Complex Form: Progress Easily with Disabled States guidance

### PR DESCRIPTION
**Pull request by finekatie**
# Summary

Update What to do and What not to do sections with new Disabled States guidance.

> [!important]
> We need to update the changelog date before merge. 

## Related issue

Affects #2246

Does not close it as there are two other PRs that will need to be created for other pages.

Related to  #2776 

## Preview link: 

Preview link:
[Progress easily preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/kf-disabledstates-progresseasily/patterns/complete-a-complex-form/progress-easily/)

## Problem statement

Current page does not have guidance for disabled states.
Research and discussion among USWDS team members have resulted in a solution

## Solution

Will be adding What to do and What not to do guidance to this page as part of a Phase 1 of Disabled States work.

## Testing and review

- Make sure copy is in the correct place
- Make sure copy has no errors or typos
- Make sure links work

## !ALERT!
I believe we'll need to add changelog - this page does not appear to have one.

### CODE REVIEW

**Comment by finekatie** [comment]
Preparing this PR today. Awaiting guidance on where the content lives in the list. Will edit the branch and request review once it's ready.

**Review by jaclinec — APPROVED** [review]
Everything looks good to me!

**Comment by amyleadem** [comment]
@finekatie 

I added a changelog for this update. I also noticed that the changelogs for the "Progress easily" ([Before](https://designsystem.digital.gov/patterns/complete-a-complex-form/progress-easily/) | [After](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/kf-disabledstates-progresseasily/patterns/complete-a-complex-form/progress-easily/)) and "Establish Trust" ([Before](https://designsystem.digital.gov/patterns/complete-a-complex-form/establish-trust/) | [After](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/kf-disabledstates-progresseasily/patterns/complete-a-complex-form/establish-trust/)) pages were not hooked up properly, so I took the liberty of fixing those up in [86e7b47](https://github.com/uswds/uswds-site/pull/2744/commits/86e7b47b403c89fbdae3b442c1c59fe953f6fb4f). 

Can you confirm that everything looks good with the changelogs on these pages?

**Review by amyleadem — APPROVED** [review]
Looks good to me!

**Comment by finekatie** [comment]
> @finekatie
> 
> I added a changelog for this update. I also noticed that the changelogs for the "Progress easily" ([Before](https://designsystem.digital.gov/patterns/complete-a-complex-form/progress-easily/) | [After](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/kf-disabledstates-progresseasily/patterns/complete-a-complex-form/progress-easily/)) and "Establish Trust" ([Before](https://designsystem.digital.gov/patterns/complete-a-complex-form/establish-trust/) | [After](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/kf-disabledstates-progresseasily/patterns/complete-a-complex-form/establish-trust/)) pages were not hooked up properly, so I took the liberty of fixing those up in [86e7b47](https://github.com/uswds/uswds-site/pull/2744/commits/86e7b47b403c89fbdae3b442c1c59fe953f6fb4f).
> 
> Can you confirm that everything looks good with the changelogs on these pages?

CONFIRMED!!!!

**Comment by finekatie** [comment]
This is ready for FFR. @thisisdano , when you have a chance, please review.

File: _data/changelogs/pattern-complex-form-progress-easily.yml [hunk 1]
```diff
@@ -2,8 +2,14 @@ title: Progress easily
 type: pattern
 changelogURL:
 items:
+  - date: NNNN-NN-NN
```
Comment (thisisdano) [inline _data/changelogs/pattern-complex-form-progress-easily.yml:5]:
```suggestion
  - date: 2024-09-09
```

**Comment by finekatie** [comment]
Hi @thisisdano , is there anything else I need to do to merge this?

File: _data/changelogs/pattern-complex-form-progress-easily.yml [hunk 2]
```diff
@@ -2,8 +2,14 @@ title: Progress easily
 type: pattern
 changelogURL:
 items:
+  - date: 2024-09-09
```
Comment (thisisdano) [inline _data/changelogs/pattern-complex-form-progress-easily.yml:5]:
```suggestion
  - date: 2024-10-04
```

File: _data/changelogs/pattern-complex-form-progress-easily.yml [hunk 3]
```diff
@@ -2,8 +2,14 @@ title: Progress easily
 type: pattern
 changelogURL:
 items:
+  - date: 2024-10-04
```
Comment (thisisdano) [inline _data/changelogs/pattern-complex-form-progress-easily.yml:5]:
```suggestion
  - date: 2024-10-15
```

**Comment by thisisdano** [comment]
Merging over a `htmlproofer` fail that I don't want to block this work any longer

**Pull request merged**